### PR TITLE
Added Course on Linked Lists

### PR DESCRIPTION
**Pull request by metabiswadeep**
## What does this PR do?
Added a course on linked lists.

### Description
Absolutely Free course to learn linked lists

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -125,6 +125,7 @@
 * [IIT Bombay Foundation of Data Structures (CS213.1x)](https://courses.edx.org/courses/course-v1:IITBombayX+CS213.1x+1T2017/course/)
 * [Intro to Data Structures and Algorithms](https://www.udacity.com/course/data-structures-and-algorithms-in-python--ud513) - Brynn Claypoole, Horatio Thomas (Udacity)
 * [JavaScript Algorithms and Data Structures](https://www.youtube.com/playlist?list=PLC3y8-rFHvwjPxNAKvZpdnsr41E0fCMMP) - Codevolution
+* [Linked Lists Course](https://www.youtube.com/watch?v=Hj_rA0dhr2I) - FreeCodeCamp
```
Comment (eshellman) [inline courses/free-courses-en.md:128]: For youtube, we prefer "playlist" links and the name of the producer/creator. https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md#platforms-and-access-notes

Comment (metabiswadeep) [reply courses/free-courses-en.md]: Firstly the link you provided doesn't work, at least for me. Also, I have edited it @eshellman

**Comment by metabiswadeep** [comment]
Please tell if any other changes are necessary before merging this PR @eshellman

File: courses/free-courses-en.md (hunk 1 above)
Comment (eshellman) [reply courses/free-courses-en.md]: whoops! https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md

File: courses/free-courses-en.md [hunk 2]
```diff
@@ -125,6 +125,7 @@
 * [IIT Bombay Foundation of Data Structures (CS213.1x)](https://courses.edx.org/courses/course-v1:IITBombayX+CS213.1x+1T2017/course/)
 * [Intro to Data Structures and Algorithms](https://www.udacity.com/course/data-structures-and-algorithms-in-python--ud513) - Brynn Claypoole, Horatio Thomas (Udacity)
 * [JavaScript Algorithms and Data Structures](https://www.youtube.com/playlist?list=PLC3y8-rFHvwjPxNAKvZpdnsr41E0fCMMP) - Codevolution
+* [Linked Lists Course](https://youtube.com/playlist?list=PLDzeHZWIZsTr54_TH_NK4ibFojS4mmQA6) - LoveBabbar
```
Comment (eshellman) [inline courses/free-courses-en.md:128]: Don't invent titles! Use the title given by the resource.
Look at other items in the list for Babbar's courses

Comment (metabiswadeep) [reply courses/free-courses-en.md]: Done. Also the other items by Babbar are not the same

File: courses/free-courses-en.md [hunk 3]
```diff
@@ -125,6 +125,7 @@
 * [IIT Bombay Foundation of Data Structures (CS213.1x)](https://courses.edx.org/courses/course-v1:IITBombayX+CS213.1x+1T2017/course/)
 * [Intro to Data Structures and Algorithms](https://www.udacity.com/course/data-structures-and-algorithms-in-python--ud513) - Brynn Claypoole, Horatio Thomas (Udacity)
 * [JavaScript Algorithms and Data Structures](https://www.youtube.com/playlist?list=PLC3y8-rFHvwjPxNAKvZpdnsr41E0fCMMP) - Codevolution
+* [Linked List - by LoveBabbar](https://youtube.com/playlist?list=PLDzeHZWIZsTr54_TH_NK4ibFojS4mmQA6) - LoveBabbar
```
Comment (eshellman) [inline courses/free-courses-en.md:128]:
```suggestion
* [Linked List - by Love Babbar](https://youtube.com/playlist?list=PLDzeHZWIZsTr54_TH_NK4ibFojS4mmQA6) - CodeHelp by Babbar
```

**Comment by metabiswadeep** [comment]
Done @eshellman

**Comment by eshellman** [comment]
please resolve conflicts.

**Pull request closed without merging**